pkg/ime_types/answers_type.go

```diff
@@ -8,4 +12,7 @@ type Answers struct {
 	KafkaClusterNamespace          string
 	KafkaClusterPersistenceEnabled bool
 	KafkaTopics                    string
+
+	KafkaQuestions     []*survey.Question
+	WorkspaceQuestions []*survey.Question
```

Comment:
I think we should not mix Questions and Answers. Maybe we can pick the question directly in the `survey` object ?